auth-api/src/auth_api/services/affiliation_invitation.py

```diff
@@ -477,12 +478,27 @@ def find_affiliation_invitation_by_id(invitation_id):
     @staticmethod
     def _get_token_confirm_path(app_url, org_name, token, query_params=None):
         """Get the config for different email types."""
-        escape_url = escape_wam_friendly_url(org_name)
-        path = f"{escape_url}/affiliationInvitation/acceptToken"
-        token_confirm_url = f"{app_url}/{path}/{token}"
+        if flags.is_on("enable-new-magic-link-formatting-with-query-params", default=False):
```

Comment:
Travis, can you add this FF to LD please?

Reply:
yes